packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeRoute, isTabletOs, routes } from '@deriv/shared';
+import LandscapeBlockerSvg from 'Assets/SvgComponents/settings/landscape-blocker.svg';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeRoute(pathname);
+    const shouldShowDtraderTabletView = pathname === routes.trade && isTabletOs;
+
+    if (is_hidden_landscape_blocker || shouldShowDtraderTabletView) return null;
+
+    return (
+        <div id='landscape_blocker' className='landscape-blocker'>
+            <div className='landscape-blocker__icon'>
+                <LandscapeBlockerSvg />
+            </div>
+            <div className='landscape-blocker__message--landscape'>
+                Please adjust your screen size for optimal viewing.
+            </div>
+            <div className='landscape-blocker__message--portrait'>
+                Please adjust your <br />
```

Comment:
Creating separate paragraphs of text using `<br>` seems to be a bad practice + it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within `<br>`s. This can be a confusing and frustrating experience for the person using the screen reader.

If it possible, it's better to use `<p>` elements, and use CSS properties to control their spacing.

Reply:
Hi @kate-deriv , br tag is semantically correct tag to add line breaker between text.